bddl/activity_definitions/make_an_egg_tomato_and_toast_breakfast/problem0.bddl

```diff
@@ -27,8 +28,9 @@
     
     (:goal 
         (and 
-            (ontop ?toast.n.01_1 ?plate.n.04_1) 
-            (ontop ?egg.n.02_1 ?toast.n.01_1) 
+            (ontop ?toast.n.01_1 ?plate.n.04_1)
```

Comment:
This task is now unfair, since there are two plates. Need to change to exists plate, probably

Reply:
Changed to a bowl.